client/state/sites/selectors/get-woo-express-trial-days-left.ts

```diff
@@ -0,0 +1,20 @@
+import { AppState } from 'calypso/types';
+import getWooExpressTrialExpiration from './get-woo-express-trial-expiration';
+
+/**
+ * Get the number of days left in the Woo Express trial. If the trial is not active, returns null.
+ *
+ * @param {AppState} state - Global state tree
+ * @returns {null|number} Number of days left in the trial, or null if the trial is not active.
+ */
+export default function getWooExpressTrialDaysLeft( state: AppState ): number | null {
+	const trialExpirationDate = getWooExpressTrialExpiration( state );
+	if ( ! trialExpirationDate ) {
+		return null;
+	}
+
+	const expiryDate = +new Date( trialExpirationDate );
+	const diffTime = expiryDate - +new Date();
```

Comment:
My main concern here is that the expiry date is almost certainly in UTC, while the user's time zone could be anything. We should handle that offset here.
Also, please explain the `+new` usage - it's not obvious to me what that is doing.

Reply:
Thanks for catching it! I started using `moment` to facilitate it. The `+new` is basically to convert it to a timestamp.